cmd/start.go

```diff
@@ -64,8 +69,13 @@ var startClientCmd = &cobra.Command{
 	Use:   "client",
 	Short: "Start the client",
 	Run: func(cmd *cobra.Command, args []string) {
+		var err error
 		config.Config.IsClient = isClient
-		config.Config.TLSParams = client.TLSBuildClientConf()
+		config.Config.TLSParams.TLSConfig, err = config.TLSBuildClientConf()
+		if err != nil {
+			log.Println(colors.Red("Unable to load TLS configuration. Shutting down."))
+			gracefulShutdown()
```

Comment:
why gratefulShutdown here ? That function wait for ctrl c to quit.

Reply:
replaced with an exit